src/spectrecoff-cli/commands/Padder.fs

```diff
@@ -45,12 +45,32 @@ type PadderDocumentation() =
         |> toConsole
 
         Many [
-            Many [
-                C "This module provides functionality from the padder widget of Spectre.Console ("
-                Link "https://spectreconsole.net/widgets/padder"
-                C ")"
+            C "This module provides functionality from the padder widget of Spectre.Console ("
+            Link "https://spectreconsole.net/widgets/padder"
+            C ")"
+            BL
+            C "Using functions from this module, any OutputPayload can be padded. The main function works similar to the one of CSS, namely it takes four padding arguments of type int as well as the element."
+            BL
+            BI [P "pad: top right bottom left (element: OutputPayload) -> OutputPayload"]
+            BL
+            C "If one does not want to set all four values differently, there are a few helpful functions that are easier to use:"
+            BL
+            BI [
+                P "padTop: (top: int) (element: OutputPayload) -> OutputPayload"
+                P "padRight: (top: int) (element: OutputPayload) -> OutputPayload"
+                P "padBottom: (top: int) (element: OutputPayload) -> OutputPayload"
```

Comment:
All the paramters here are called top and should instead match the method name :)

Reply:
Correct. Done.